sdk/dotnet/examples/CSharpExamples/CSharpExamples.csproj

```diff
@@ -0,0 +1,13 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
We probably want to erase all examples before merging and move them to examples repo, per-cloud repos, and templates.

Reply:
can i leave that to you @mikhailshilkov ?

Reply:
I'm keeping them so that you could test any changes. We should remove them either right before merging, or whenever the Azure provider is in place and we have some examples in that repo.